packages/proxy/lib/http/util/csp-header.ts

```diff
@@ -107,7 +107,9 @@ export function parseCspHeaders (headers: OutgoingHttpHeaders, headerName: strin
       policy = cspRegExp.exec(cspHeader)
     }
 
-    return [...acc, policies]
+    acc.push(policies)
```

Comment:
Do we have any data besides theoretical Big O notation that the change is impactful enough to warrant change? Either way, the change seems fine to me!